Carrousel tests (#3)
wolox-training/aq-java-appium: README.md

    testCompile group: 'io.cucumber', name: 'cucumber-junit', version: '6.9.1'
}
```

## Mobile Swipe

> To perform the action horizontal and vertical scroll down to an element we can guide the : http://appium.io/docs/en/writing-running-appium/tutorial/swipe/android-layout-direction/

Reviewer: ```suggestion
To perform an horizontal and vertical scroll down to an element we can follow this guide: http://appium.io/docs/en/writing-running-appium/tutorial/swipe/android-layout-direction/
```
